### DISCUSSION TITLE
View rejected emails

### DISCUSSION THREAD

**Discussion by brinza888** [Mailu/Mailu#3169] [User support]
Well, Rspamd sometimes rejects emails with .xlsm attachment. This is fine in usual cases, but not when you trust sender. So there is a question, how can I view those emails? Is there something like quarantine directory?

Symbols on those rejected emails are OLETOOLS_MACRO_SUSPICIOUS, MIME_BAD_ATTACHMENT and OLETOOLS.

Thank you in advance!

**Comment by ghostwheel42** [comment]
What do you mean by "view email"?

You can see which emails were rejected in the rspamd admin panel (time, sender, recipient, subject) and in the log (time, sender recipient). 

You can't see the email's content, as it has been rejected and not delivered.

There's no quarantine feature.

**Reply by brinza888** [reply]
Yeah, quarantine feature is what I need in such situations.
Is it completely impossible to implement for this postfix and rspamd setup?

**Reply by nextgens** [reply]
It's not impossible it's just that we have not implemented it in Mailu.

File a feature request and it may happen.
